.gitignore

```diff
@@ -0,0 +1 @@
+.DS_Store
```

Comment:
whats up with this file?

Reply:
It's something MacOS Finder creates for some reason. We don't want it committed to the repo.

Reply:
oh i see. In that case thats fine.